pkgs/applications/editors/pulsar/default.nix

```diff
@@ -136,8 +136,6 @@ stdenv.mkDerivation rec {
     patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
       --set-rpath "${newLibpath}" \
       $opt/resources/app/ppm/bin/node
-    patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
-      $opt/resources/app.asar.unpacked/node_modules/symbols-view/vendor/ctags-linux
```

Comment:
I see `opt/Pulsar/resources/app.asar.unpacked/node_modules/symbol-provider-ctags/vendor/ctags-linux` exists, should it be patchelf'ed?

Reply:
Thank you for the review!

> I see `opt/Pulsar/resources/app.asar.unpacked/node_modules/symbol-provider-ctags/vendor/ctags-linux` exists, should it be patchelf'ed?

Indeed I seem to have missed this... I think it would be better to still patch it at the new location; let me address this in a follow-up commit. It's easy for me to miss these kinds of issues as I am not running NixOS but nix as a package manager on an FHS, so the bug might be hidden from me haha